Project-1/README.md

```diff
@@ -38,29 +102,234 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
+a) B that says that x is a brother of y
+
+Per (Person) ≡ M ⊔ ¬M
+p2 (Parent of 2) ≡ ∃parentof. ≥ 2 ⊓ ¬(∃parentof. = 1) -- assuming gender is assigned based on chromosome 
+B ≡ M ⊓ ∃p2¯.Per
+
+(b) A that says that x is an aunt of y
+
+∃p2¯.parentof.parentof.Per
+A ≡ ¬M ⊓ (∃p2¯. (parentof. ≥ 2).(parentof.Per)) ⊔ (¬parentof. Per ⊓ ∃p2¯.parentof.parentof.Per)
+U(uncle) ≡ M ⊓ (∃p2¯. (parentof. ≥ 2).(parentof.Per)) ⊔ ¬parentof.Per
+
+(c) C that says that x and y are cousins
+
+gp2(Grandparent with at least two children which each have children) ≡ ∃parentof ≥ 2.parentof.Per
+C ≡ gp2¯.per
+Cannot define "being cousins" in description logic.
+
+(d) O that says that x is an only child
+O ≡ ¬ ∃p2¯.Per
+
+(e) T that says that x has exactly two brothers
+This cannot be done 
+
+5. Select two formulas defined in ALC from question 4 to form the basis of a T-Box. Supplement this T-box with whatever other axioms you like, as well as an A-box, so that you ultimately construct a knowledge base K = (T,A). Provide a _model_ of K. This may be graphical or symbolic or both.
+
+ TBox = {Brother ⊆ Male,		(TBox.1)
+	Aunt ⊆ ~Male}		(TBox.2)
+
+Donnie = {Frank: Brother ∩ Male		(Axiom.1)
+	Remi: Brother ∩ Male		(Axiom.2)
+	Mary: Aunt			(Axiom.3)
+	(Donnie, Frank): parent_of	(Axiom.4)
+	(Donnie, Remi): parent_of}		(Axiom.5)
+	
+ΔI = {w, x, y, z},
+Donnie = w
+Remi = x
+Mary = y
+Frank = z
+Male = {w, x}
+Brother = {w, x}
+Aunt = {y}
+Parent_of = {(z,w), (z,x)}
 
 6. Explain the difference - using natural language - between the first-order prefixes:
-  ```(a) ∃x∀y and ∀x∃y
+ ```(a) ∃x∀y and ∀x∃y
   (b) ∃x∀y∃z and ∀x∃y∀z 
   (c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
-```
-	
+
+```(a) ∃x∀y says something like there is some x such that for all y
+(a) ∀x∃y says for all x there is some y 
+(b) ∃x∀y∃z says there is some x such that for all y there is some z such that 
+(b) ∀x∃y∀z says for all x there is come y such that for all x
+(c) ∀x∃y∀z∃w says for all x there is some y such that for all z there is some w
+(c) ∃x∀y∃z∀w says there is some x such that for all y there exists some z such that for all w
+
 7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
 ```
 ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
 ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
-```
+
+``` Here is my diagram of the universe: [label](file:///Universe.pub)
+U: {0,1}
+R: {(0,0), (0,1), (1,1)} 
 	
 8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
-  ```(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+
+  ````(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
   (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
   (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
   (d) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
-```
-	
+
+  (a)¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
```

Comment:
What rule justifies moving from ¬∀x∀y to ¬∀y without first removing the negation?

Reply:
This is a great point. I used the tree deviation generator, I know in a derivation the rule to get rid of the negation of a universal is to use the quantifier negation. However, this would produce ∃x¬∀y(¬(Px  ∧ Qx) → (¬Px v (¬Qx))
this would completely change how the derivation happens as to get rid of the existential quantifier you have to instantiate all instances of the variable x to a new variable, so too to get rid of the negation that now appears in form of ¬∀y. This would completely change the derivation as a whole. I am looking into this as I find it odd the tree-proof generator would just force this. Hold tight for more info :)

Reply:
@johnbeve I put this derivation back into https://www.umsu.de/trees/ 
and it is giving the exact same derivation by negating the whole thing. Clearly, this is an indirect derivation but as mentioned above to clear this negation one should first convert the universal to an existential. This was my original gripe with this system, they give no annotations for how to move through. I will await your comment on this but perhaps I should just do my own tree derivation?

Reply:
> and it is giving the exact same derivation by negating the whole thing. Clearly, this is an indirect derivation but as mentioned above to clear this negation one should first convert the universal to an existential. This was my original gripe with this system, they give no annotations for how to move through. I will await your comment on this but perhaps I should just do my own tree derivation?

In order to fix this issue, I have to rewrite all tree proofs manully...

Reply:
@peihongx Thank you for your input. I better get cracking lol